### PR TITLE
Blend search results.

### PR DESCRIPTION
**Pull request by chuckharmston**
For Orlando, we're going to blend using a set of fixed slots: 
- Tiny Machine
- Top navigation suggestion
- Top search suggestion
- Navigation suggestion
- Navigation suggestion
- Navigation suggestion
- Search suggestion
- Search suggestion
- Search suggestion

It's not pretty, but it will be moderately effective. In this branch I'm working toward doing so.

### CODE REVIEW

**Comment by chuckharmston** [comment]
I also went `s/lilmac/LittleMachine/` out of embarrassment that it took me 10 minutes of poking around to figure out why we were doing something special for Mac Minis.

File: src/views/little_machine/little_machine_index_view.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import LittleMachineIndexTemplate from '../../templates/little_machine/index.html';
+import LittleMachineItemView from './little_machine_item_view';
+import LittleMachineResults from '../../collections/little_machine_results';
+import LittleMachineAdapter from '../../adapters/little_machine_adapter';
+
+export default BaseView.extend({
+  template: LittleMachineIndexTemplate,
+
+  initialize () {
+    this.little_machine = LittleMachineAdapter;
```
Comment (pdehaan) [inline src/views/little_machine/little_machine_index_view.js]:
```
src/views/little_machine/little_machine_index_view.js
  11:9  error      Identifier 'little_machine' is not in camel case      camelcase
```

I think ESLint is complaining that this isn't `this.littleMachine`.

Comment (chuckharmston) [inline src/views/little_machine/little_machine_index_view.js]: Yeah, I'm ignoring ESLint for now as I shuffle things around. I'll make sure it's not complaining before I remove the [wip].

**Comment by chuckharmston** [comment]
screen shot 2015-11-18 at 9 53 12 am

Results are now blended.

File: src/views/search_suggestions/search_suggestions_top_view.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import BaseView from '../base_view';
```
Comment (chuckharmston) [inline src/views/search_suggestions/search_suggestions_top_view.js:1]: There's a lot of boilerplate in these views. Can't help but feel like we could do better using inheritance.

**Comment by chuckharmston** [comment]
I think this is ready for review.

r? @6a68

**Comment by chuckharmston** [comment]
(I'll flatten the commits before merging)

File: src/views/search_suggestions/search_suggestions_top_view.js (hunk 2 above)
Comment (jaredhirsch) [inline src/views/search_suggestions/search_suggestions_top_view.js:1]: Yeah, I totally agree. Looking forward to cleaning this up post-mozlando

**Comment by jaredhirsch** [comment]
Sorry for the delay: I was having trouble getting it working locally, had typoed one of the prefs :-P

Looks good overall, comments incoming

File: src/views/activity/activity_top_view.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  template: ActivityIndexTemplate,
+
+  initialize () {
+    this.adapter = activityResultsAdapter;
+    this.listenTo(this.adapter.results, 'reset', this.render);
+  },
+
+  afterRender () {
+    this.removeSubviews();
+    let items = this.sliceCollection(this.adapter.results, ActivityResultsCollection, 0, 1);
```
Comment (jaredhirsch) [inline src/views/activity/activity_top_view.js:17]: I'm not a fan of this "copy a model into a singleton collection, then render this random collection" approach, but it's consistent with what was there, so let's worry about cleanup post-mozlando :beers:

**Comment by jaredhirsch** [comment]
@chuckharmston Nice work! Merging

**Pull request merged**